marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/WriteDataTest.java

```diff
@@ -48,6 +47,25 @@ void ingestThreeFruitsWithBatchSizeOfTwo() throws IOException {
         verifyFruitCount(3, "The commit call should result in the 3rd fruit being ingested");
     }
 
+
+    @Test
+    public void testBulkIngestWithoutUriPrefix() throws IOException {
+        DataWriter<InternalRow> dataWriter = buildDataWriter("1", "");
+        dataWriter.write(buildRow("pineapple", "green"));
+
+        String uriQuery = "cts.uris('', null, cts.andQuery([\n" +
+            "  cts.jsonPropertyValueQuery('fruitName', 'pineapple')\n" +
+            "]))";
+
+        EvalResultIterator uriQueryResult = getHubClient().getStagingClient().newServerEval().javascript(uriQuery).eval();
+        assertTrue(uriQueryResult.hasNext());
+        String uri = uriQueryResult.next().getString();
+
+        assertTrue(uri.endsWith(".json"));
+        assertFalse(uri.startsWith("/"));
```

Comment:
Assertion calls are often the best place to document *why* something should be the way it is. In this case, there's a question of - well why doesn't it start with "/"? So I'd do:

    assertFalse(uri.startsWith("/"), "If the user wants the URI to start with a forward slash, the user must provide one. If the user doesn't, then it's assumed that the user doesn't want a forward slash at the start of the URI, so the endpoint will not add one automatically.");

That makes it very clear why it's expected. This isn't needed for every assertion, but this is a good one to document in the above manner.